kong/tools/config_loader.lua

```diff
@@ -182,7 +182,10 @@ function _M.load(config_path)
     config.nginx_working_dir = fs.current_dir().."/"..config.nginx_working_dir
   end
 
-  config.plugins = utils.table_merge(constants.PLUGINS_AVAILABLE, config.custom_plugins)
+  config.plugins = constants.PLUGINS_AVAILABLE
+  for _, v in ipairs(config.custom_plugins) do
+    table.insert(config.plugins, v)
+  end
```

Comment:
The original code had 3 tables; `config.plugins`, `constants.PLUGINS_AVAILABLE` and `config.custom_plugins`

The new code has only 2; `config.plugins` and `config.custom_plugins`, the one from `constants.PLUGINS_AVAILABLE` just points to the `config.plugins` one. So effectively you're copying the custom plugins into the constants table, which seems odd imo, why would it be a constant then?

Or is this intentional?